fuseops/ops.go

```diff
@@ -150,6 +150,8 @@ type SetInodeAttributesOp struct {
 	Mode  *os.FileMode
 	Atime *time.Time
 	Mtime *time.Time
+	Uid   *uint32
```

Comment:
Is there a reason why you’re adding Uid/Gid only to some operations, but not to all? Isn’t the uid/gid field set by the kernel for all requests?